pkg/local_object_storage/blobstor/control.go

```diff
@@ -10,11 +10,9 @@ import (
 func (b *BlobStor) Open(readOnly bool) error {
 	b.log.Debug("opening...")
 
-	for i := range b.storage {
-		err := b.storage[i].Storage.Open(readOnly)
-		if err != nil {
-			return fmt.Errorf("open substorage %s: %w", b.storage[i].Storage.Type(), err)
-		}
+	err := b.storage.Storage.Open(readOnly)
```

Comment:
You can just `return b.storage.Storage.Open()` because there is no concept of substorage any more and additional error data is meaningless.